content/ruby/concepts/scope/scope.md

```diff
@@ -0,0 +1,100 @@
+---
+Title: "Scope"
+Subjects:
+  - "Web Development"
+  - "Computer Science"
+Tags: 
+  - "Scope"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-ruby"
+  - "https://www.codecademy.com/learn/paths/web-development"
+---
+
+In Ruby, variables have a scope that outlines what other variables and methods are available to them versus which ones are not. There are different levels of scope: 
+
+* Class-level
+* Instance-level
+* Global-level
+* Local-level
+
+## Class-level 
+
+Variables defined at the class-level scope are usually marked with `@@`. They are available for use anywhere within the class block or any sub-class blocks: 
+
+```rb
+class MyClass 
+  @@class_variable = "This is the Class variable"
+
+  def inner_method 
+    "From MyClass: #{@@class_variable}"
+  end
+end
+
+class SubClass < MyClass
+  def display_class_variable 
+    "From SubClass: #{@@class_variable}"
+  end
+end
+
+
+classInstance = MyClass.new() 
+
+subClassInstance = SubClass.new()
+
+puts subClassInstance.display_class_variable # Output: From SubClass: This is the Class variable
```

Comment:
This comment and the next one are pretty long. Potentially moving them onto their own line:

```ruby
...

puts subClassInstance.display_class_variable 
# Output: From SubClass: This is the Class variable

puts classInstance.inner_method 
# Output: From MyClass: This is the Class variable
```